source/plugin/permissions.rst

```diff
@@ -153,9 +153,9 @@ Otherwise it will be inherited from any parent ``Subject``. It does not matter w
 
 If neither the subject itself nor any parent subjects grant or deny a permission then it will be inherited from the
 default ``Subjects``. Each ``SubjectCollection`` defines its own defaults. The global and weakest default subject can be
-obtained from the ``PermissionService``. It is recommended that plugins define their own permissions to the default's
-transient ``SubjectData`` during every server start-up. This allows the server owner to overwrite the defaults defined
-by plugins according to his needs using the default's persistent ``SubjectData``.
+obtained from the ``PermissionService``. It is recommended (but not mandatory) that plugins define their own permissions
```

Comment:
@zml2008 Would you mind giving your opinions here? It's not very clear either way, and I think both sides are pretty valid options. 

In attempt to tl;dr, it's basically: plugins should declare default permissions vs server admins should have to add them manually.